# HG changeset patch
# User Rik <rik@octave.org>
# Date 1432526862 25200
#      Sun May 24 21:07:42 2015 -0700
# Node ID cd5ae0f080e8e63da1a5ae7e6d84ed1e356c0d61
# Parent  ccef60b2a05869c749204de6669546869f701e48
maint: Remove interp1q deprecated in 3.8.

* scripts/deprecated/module.mk: Remove interp1q.m from build system.

diff --git a/scripts/deprecated/module.mk b/scripts/deprecated/module.mk
--- a/scripts/deprecated/module.mk
+++ b/scripts/deprecated/module.mk
@@ -4,17 +4,16 @@ deprecated_FCN_FILES = \
   deprecated/bicubic.m \
   deprecated/delaunay3.m \
   deprecated/dump_prefs.m \
   deprecated/find_dir_in_path.m \
   deprecated/finite.m \
   deprecated/fmod.m \
   deprecated/fnmatch.m \
   deprecated/gmap40.m \
-  deprecated/interp1q.m \
   deprecated/isstr.m \
   deprecated/loadaudio.m \
   deprecated/luinc.m \
   deprecated/mouse_wheel_zoom.m \
   deprecated/nfields.m \
   deprecated/octave_tmp_file_name.m \
   deprecated/playaudio.m \
   deprecated/saveaudio.m \
